nodeJS/express_and_mongoose/deployment.md

```diff
@@ -26,7 +26,7 @@ You've already had some experience using a hosting provider when you deployed pr
 
 Static websites consist of pre-written HTML pages. They are "static" because everyone who visits them will see the same content. To build static sites, you only need HTML, CSS and JavaScript.
 
-Dynamic websites, on the other hand, are websites that can change content based on the user who is visiting them. Twitter is a good example; every Twitter user sees different content on their homepage feed based on who they follow. To build dynamic sites, you still need HTML, CSS and JS. But additionally, you need a server-side application and a database.
+Dynamic websites, on the other hand, are websites that can change content based on the user who is visiting them. X is a good example; every user sees different content on their homepage feed based on who they follow. To build dynamic sites, you still need HTML, CSS and JS. But additionally, you need a server-side application and a database.
```

Comment:
Nit: without sufficient context, "X" looks like a stand in we forgot to put something in. At least for now. I think most occurrences of X would be more readable and understandable with the "(formerly known as Twitter)" note.
 
```suggestion
Dynamic websites, on the other hand, are websites that can change content based on the user who is visiting them. X (formerly known as Twitter) is a good example; every user sees different content on their homepage feed based on who they follow. To build dynamic sites, you still need HTML, CSS and JS. But additionally, you need a server-side application and a database.
```